tests/gitlab_api_mock.py

```diff
@@ -168,7 +166,7 @@ def add_user(self, info, is_current=False, sudo=None, from_state=None, to_state=
     def add_project(self, info, sudo=None, from_state=None, to_state=None):
         self.add_resource('/projects/{0.id}', info, sudo, from_state, to_state)
         self.add_transition(
-            GET('/projects/{0.id}/merge_requests'.format(attrs(info))),
+            GET(f'/projects/{attrs(info)}/merge_requests'),
```

Comment:
Here's another broken one

```suggestion
            GET(f'/projects/{attrs(info).id}/merge_requests'),
```